plugin/etcd/xfr.go

```diff
@@ -1,25 +1,17 @@
 package etcd
 
 import (
-	"context"
 	"time"
 
 	"github.com/coredns/coredns/request"
-
-	"github.com/miekg/dns"
 )
 
-// Serial implements the Transferer interface.
+// Serial returns the serial number to use.
 func (e *Etcd) Serial(state request.Request) uint32 {
 	return uint32(time.Now().Unix())
 }
 
-// MinTTL implements the Transferer interface.
+// MinTTL returns the minimal TTL.
```

Comment:
I know this is just comment docs cleanup, but is there any documentation on why this is the way it is?

Reply:
you mean why '30'? I can't recall, maybe SkyDNS did this